### PR TITLE
Development

### PR DESCRIPTION
**Pull request by BalintMendli**

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -0,0 +1,128 @@
+body {
+  padding-left: 0px;
+  padding-right: 0px;
+  margin: 0;
+  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif';
```
Comment (zlayabekrija) [inline style.css]: sans-serif should be within ''

File: index.html [hunk 2]
```diff
@@ -8,11 +8,143 @@
...
+      &nbsp; &nbsp; Watcher
+      <ul>
+        <li><a href="#">Home</a></li>
+        <li><a href="#">News</a></li>
+        <li><a href="#">Contact</a></li>
+        <li><a href="#">About</a></li>
+      </ul>
+    </nav>
+    <div class="video">
+      <iframe
```
Comment (zlayabekrija) [inline index.html]: iframe should use digits not %

File: index.html [hunk 3]
```diff
@@ -8,11 +8,143 @@
...
+      ></span>
+      <img src="img/share.png" width="20px" alt="share" />
+      <span>&nbsp;SHARE&nbsp;&nbsp;&nbsp;</span>
+      <img src="img/save.png" width="20px" alt="share" />
+      <span>&nbsp;Save</span>
+    </div>
     <div class="lower">
-      <div class="other-videos">Other videos</div>
-      <div class="comments">Comments</div>
+      <div class="other-videos">
```
Comment (zlayabekrija) [inline index.html]: Other videos should be on the right side of the main video, please move them accordingly.
Comments should be underneath the video.

File: index.html [hunk 4]
```diff
@@ -8,11 +8,143 @@
...
+          <img src="img/photo.jpg" width="45px" alt="thumbnail" class="photo" />
+          <div class="comment-text">
+            <b>Commenter</b>
+            <div>Comment</div>
+          </div>
+        </div>
+      </div>
     </div>
+    <script>
+      (function(d, e, s) {
```
Comment (zlayabekrija) [inline index.html]: js should go to script.js file or similar.

File: index.html [hunk 5]
```diff
@@ -8,11 +8,143 @@
     <link rel="stylesheet" type="text/css" media="screen" href="style.css" />
   </head>
   <body>
-    <nav class="navbar">Watcher</nav>
-    <div class="video">Video</div>
+    <nav class="navbar">
```
Comment (zlayabekrija) [inline index.html:16]: Please redo the navbar so it looks more youtube-like. Use fontawsome for icons if you like, and add the search field.

**Review by zlayabekrija — CHANGES_REQUESTED** [review]
This was excellent first try guys (especially the song). There are things to be changed.
Refrain yourself from doing inline styling, use css, also I see a lot of repetition of code (image width), make classes instead. 
Please before submitting your code, run https://validator.w3.org/ to validate your html and css. Also, use https://htmlpreview.github.io/ to preview your page on the live server. There can be a lot of differences between local and live preview.

After you are done with these changes, please submit another review request.

**Comment by AlejoCode** [comment]
All improves taked into account, please review all the new changes.

Thank you so much @zlayabekrija zlayabekrija

File: style.css [hunk 6]
```diff
@@ -0,0 +1,194 @@
...
+}
+
+.left {
+  flex: 1;
+}
+
+.video {
+    position: relative;
+    overflow: hidden;
+    padding-top: 56.25%;
```
Comment (zlayabekrija) [inline style.css]: too big padding please adjust it.

File: style.css [hunk 7]
```diff
@@ -0,0 +1,194 @@
...
+}
+
+.respIframe {
+    position: absolute;
```
Comment (zlayabekrija) [inline style.css]: unnecessary remove it please

File: style.css [hunk 8]
```diff
@@ -0,0 +1,194 @@
...
+    top: 0;
```
Comment (zlayabekrija) [inline style.css]: unnecessary

File: style.css [hunk 9]
```diff
@@ -0,0 +1,194 @@
...
+    left: 0;
+    width: 100%;
```
Comment (zlayabekrija) [inline style.css]: unnecessary

File: style.css [hunk 10]
```diff
@@ -0,0 +1,194 @@
...
+    height: 100%;
```
Comment (zlayabekrija) [inline style.css]: unnecessary

**Review by zlayabekrija — CHANGES_REQUESTED** [review]
Please adjust required changes .

**Comment by BalintMendli** [comment]
We did the requested changes. We had to use 100% for the video container to give it responsiveness.
Please let us know if it is good or should we use another solution.
Thank you.

**Review by zlayabekrija — APPROVED** [review]
Thumbs up from me 👍

**Pull request merged**